src/Playwright/Transport/Connection.cs

```diff
@@ -201,6 +209,11 @@ internal JsonSerializerOptions GetDefaultJsonSerializerOptions()
             return options;
         }
 
+        internal Task<PlaywrightImpl> InitializePlaywrightAsync()
+        {
+            return _rootObject.InitializeAsync();
+        }
+
         internal async Task<T> WaitForObjectWithKnownNameAsync<T>(string guid)
```

Comment:
Keep in mind to do the same for BrowserType.Connect and then you can remove the `WaitForObjectWithKnownNameAsync` method.

Reply:
Yes, I'm working on that one now.